test/integration/html/index.html

```diff
@@ -11,5 +11,6 @@ <h1>Hello world</h1>
   <a href="tel:+33636757575">Tel link</a>
   <script src="index.js"></script>
   <script src="https://unpkg.com/parcel-bundler"></script>
+  <i>hello</i> <i>world</i>
```

Comment:
To elaborate on this scenario, since you wouldn't write the above html in practice, consider the following:

```html
<div>
<span class="a">Hello in style a</span> <span class="b">World in style b</span>
</div>
```

Reply:
@brandon93s Should I update my PR ? Or is this just for info ?

Reply:
Just info for reviewers.

Reply:
hmm, wonder why this is the default behavior for htmlnano? wonder if we should bring this up with them.